### ISSUE TITLE
Usage with yarn berry fails

### ISSUE THREAD

**Issue by faustbrian** [paritytech/txwrapper#382]
I'm trying to use this package with yarn berry but am running into an odd issue relating to the below log.

```
    Qualified path resolution failed - none of the candidates can be found on the disk.

    Source path: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/lib/index.js
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/lib/index.js.ts
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/lib/index.js.tsx
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/lib/index.js.js
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/lib/index.js.jsx
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/lib/index.js.json
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/lib/index.js.node
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper.ts
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper.tsx
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper.js
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper.jsx
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper.json
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper.node
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/index.ts
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/index.tsx
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/index.js
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/index.jsx
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/index.json
    Rejected candidate: /Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip/node_modules/@substrate/txwrapper/index.node
```

When unpacking the file `/Users/brian/Work/platform-sdk/.yarn/cache/@substrate-txwrapper-npm-3.10.7-5c033123a2-2.zip` it only contains the package.json, README and LICENSE but is missing the `lib` folder but I'm not sure why it would be missing as I unpacked a dozen other packages and they are all fine.

**Comment by anhcao142** [comment]
I also get this problem when trying to upgrade `@substrate/txwrapper` to `3.10.7`

**Comment by emostov** [comment]
Fixed with `3.10.8`. Please reopen this issue if you are still having the same problem with `3.10.8`.
